src/Tests_TestUtils.c

```diff
@@ -30,4 +30,32 @@ BOOL Ros_Testing_CompareLong(long a, long b)
     return ok;
 }
 
+BOOL Ros_Testing_CompareINT64(INT64 a, INT64 b)
```

Comment:
Should we rename this to indicate it actually checks for *equality*?

I realise the existing functions follow a similar naming scheme, but we can take this opportunity to introduce more aptly named functions already (and address renaming the others at a later time).